### DISCUSSION TITLE
Words spacing in paragraph using tailwind CSS

### DISCUSSION THREAD

**Discussion by MinalPatilRPA** [tailwindlabs/tailwindcss#13062] [Help]
Hi Team,

I have shared image of paragraph  as shown in below. I want set space between the words, but as shown in fig. space between the words are different and wide . I have set space as normal and text alignment is justify. 

![image](https://github.com/tailwindlabs/tailwindcss/assets/122990718/c038774a-6157-4d71-81cd-3bbb7d0f41d1)

my code:
/*
<p className="text-black/75 text-xl text-justify  " >
               Software Developer with over 2 years of experience working with multinational clients. Possesses excellent technical knowledge of technologies with exposure to other automation solutions & various business processes, ERP solution automations. Excellent written and verbal communication along with great interpersonal skills.
              </p>
*/

Thanks 
Minal P

**Comment by wongjn** [comment]
If you don't want the wider spacing between words, consider removing the justified text alignment.

**Reply by MinalPatilRPA** [reply]
But I also want to set justified text alignment.
![image](https://github.com/tailwindlabs/tailwindcss/assets/122990718/d872f4f3-6cdd-4ef1-8ca0-3b08532e24e5)

**Reply by wongjn** [reply]
You'd have to choose one or the other. It's precisely the justified text alignment that is causing the inconsistent word spacing. The words are spaced out precisely so that the text can appear justified.

**Reply by MinalPatilRPA** [reply]
ok Thanks

**Reply by MinalPatilRPA** [reply]
Any other solution to solve this issue.
